code/modules/cargo/packs.dm

```diff
@@ -156,6 +156,7 @@ datum/supply_pack
 
 /datum/supply_pack/emergency/syndicate/fill(obj/structure/closet/crate/C)
 	var/crate_value = 50
+	var/list/uplink_items = get_uplink_items(ticker.mode)
```

Comment:
There's a edge case runtime that occurs if an uplink hasn't been opened yet. It tries to pick from an empty list because the global list isn't filled.
The other ones do it the same way.